pkg/trace/stats/quantile/converter_test.go

```diff
@@ -127,3 +143,7 @@ func TestDDToGKSketch(t *testing.T) {
 	t.Run("constant10", testDDSketchToGKConstant(10))
 	t.Run("constant1e3", testDDSketchToGKConstant(1000))
 }
+
+func TestDecoding(t *testing.T) {
+	t.Run("dense_sketch_from_Java", testUnmarshalDenseSketchFromFile("java_generated_dense_ddsketch.ddsketch", 101, 631))
+}
```

Comment:
Can we put the binary file in `pkg/trace/test/fixtures` ? You'll have to refer to it as `./pkg/trace/test/fixtures/java_genera.....` because tests are always run from the root of the repository in CI.

Reply:
ok

Reply:
Friday is freeze. I should either merge this or move it to the next version milestone. All that's left is this comment with moving the file to that new folder. If you can do it sometime this week I'm happy to merge it.